packages/core/src/codewhisperer/activation.ts

```diff
@@ -220,7 +226,7 @@ export async function activate(context: ExtContext): Promise<void> {
         /**
          * On recommendation acceptance
          */
-        acceptSuggestion.register(context),
+        acceptSuggestion.register(context, container),
```

Comment:
any advice on this? 
another way to do it is just call `Container.instance` within the commands